tcf_website/templates/course/course_professor.html

```diff
@@ -61,7 +61,7 @@ <h2 class="mr-md-3">{{ course.title }}</h2>
                 <div class="ratings-row">
                     <div class="rating-card">
                         <span class="rating-card-num rating-num">&mdash;</span>
-                        <p class="rating-card-desc"><i class="fa fa-star fa-fw"></i>&nbsp;Rating</p>
+                        <p class="rating-card-desc"><i class="fa fa-star fa-fw"></i>&nbsp;{{ num_reviews }} {% if num_reviews == 1 %} Review {% else %} Reviews {% endif %}</p>
```

Comment:
Since we already add the number of comments on line 196, wouldn't it be better to just keep this as Rating?

Reply:
I still think this should be kept as rating and we can move number of reviews somewhere else